.gitallowed

```diff
@@ -0,0 +1,4 @@
+const TRACE_EXCLUSIONS =
+'argsToMask' => \[ 'password', \],
+
+.gitallowed
```

Comment:
**New file**

Added to prevent git secrets blocking commits for these false positives